mountinfo/mountinfo_linux.go

```diff
@@ -11,6 +11,14 @@ import (
 	"strings"
 )
 
+// GetMountsFromReader retrieves a list of mounts from the
+// reader provided, with an optional filter applied (use nil
+// for no filter). This can be useful in tests or benchmarks
+// that provide a fake mountinfo data.
+func GetMountsFromReader(reader io.Reader, f FilterFunc) ([]*Info, error) {
```

Comment:
Now that GetMountsFromReader is effectively just an alias for parseInfoFile, should we just export parseInfoFile on Linux instead?

Looks like it's not used anywhere currently https://grep.app/search?q=mountinfo.GetMountsFromReader

Reply:
It is used in a few places, so I'd rather not change the name. Surely we can just rename `parseInfoFile` to `GetMountsFromReader`, if this is what you mean

One such place is https://github.com/opencontainers/runc/pull/2599/commits/999ff3f9dbee376bf1db8abe3fedd9594a7ef582, and another is

Reply:
... and another is https://github.com/moby/moby/pull/40694

Reply:
Nope, it's not it. I don't remember why I added it but I definitely had a good reason to :-)

Reply:
But surely it makes sense to make it Linux-only

Reply:
Please see https://github.com/moby/sys/pull/39